ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -40,8 +40,7 @@ let stunnel_m = Mutex.create ()
 let restart_stunnel_nomutex ~__context ~accept =
 	info "Restarting stunnel (accepting connections on %s)" accept;
 	let back_compat ~__context =
-		let localhost = Helpers.get_localhost ~__context in
-		if Db.Host.get_ssl_legacy ~__context ~self:localhost
+		if Stunnel.is_legacy_protocol_and_ciphersuites_allowed ()
```

Comment:
In the light of this change, a string in xapi.ml must be changed too: in the line
`"Config (from file) for outgoing stunnels", [], set_stunnel_legacy_inv;`
the word "outgoing" is no longer valid since it controls both outgoing and incoming now.